### PR TITLE
[MI-790] expand collapse

### PR DESCRIPTION
**Pull request by joseconsador**
/cc @zendesk/mintegrations

### Description

Toggles the order display when clicking on the icon.

**On**
screen shot 2016-09-08 at 5 09 31 pm

**Off**
screen shot 2016-09-08 at 5 09 23 pm

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-790

### Risks

### CODE REVIEW

File: src/templates/customer.hdbs [hunk 1]
```diff
@@ -14,3 +14,8 @@
 {{/if}}
 
 <section data-orders></section>
+<div class="u-mt-sm">
+	<svg id="orders-toggle">
```
Comment (samgavinio) [inline src/templates/customer.hdbs]: Seems to be some weird tabbing here.

**Comment by miogalang** [comment]
please rebase off v2 so it is mergable

**Comment by samgavinio** [comment]
👍

File: src/javascripts/shopify_app.js [hunk 2]
```diff
@@ -237,6 +245,29 @@ var ShopifyApp = {
       return text.substr(0, this.noteCharacterLimit) + '...';
     }
     return text;
+  },
+
+  loadSvg: function() {
```
Comment (miogalang) [inline src/javascripts/shopify_app.js]: What is an Svg? Can this be more descriptive to normal people?

Comment (joseconsador) [inline src/javascripts/shopify_app.js]: Svg is the image format used by garden to display sprites.

Comment (miogalang) [inline src/javascripts/shopify_app.js]: Maybe it should be `loadSprites`?

**Comment by joseconsador** [comment]
@miogalang 💇

**Comment by miogalang** [comment]
👍

**Pull request merged**